### PR TITLE
New package: yadm-1.10.0

### PR DESCRIPTION
**Pull request by NeoChontrix**

### CODE REVIEW

File: srcpkgs/yadm/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Yet Another Dotfiles Manager"
+maintainer="John <johnz@posteo.net>"
+license="GPL-3"
+homepage="https://thelocehiliosan.github.io/yadm/"
+distfiles="https://github.com/TheLocehiliosan/${pkgname}/archive/${version}.tar.gz"
+checksum=3f77b51197fe14d24f670dbc3e5edc4ae329fdd91c03db297ab929e7bc1168bb
+
+do_build() {
+	make yadm.md
+}
```
Comment (Vaelatern) [inline srcpkgs/yadm/template]: Please use a build style.

File: srcpkgs/yadm/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'yadm'
+pkgname=yadm
+version=1.10.0
+revision=1
+hostmakedepends="make"
```
Comment (Vaelatern) [inline srcpkgs/yadm/template]: Not necessary.

**Comment by Vaelatern** [comment]
Not yet ready, revisit after a build style is used.

**Comment by Vaelatern** [comment]
@NeoChontrix  please do not close this PR.
Please do not add new commits.
What you want instead is to "squash" commits, or "fixup" your commit.
Then `push -f` to force an update to this PR.

**Comment by NeoChontrix** [comment]
Fixed it and removed the unnecessary make stuff that I only had in for testing.

**Comment by NeoChontrix** [comment]
Or not, what would be the build type for just extracting?

**Comment by Vaelatern** [comment]
OK except...
should this be a package? It's a manpage and shell script. I'm not comfortable saying absolutely no, but I'd like someone else's feedback before merging.

**Comment by the-maldridge** [comment]
We've packaged less useful things of equivalent complexity in the past, though I personally like to not rely on root access to bootstrap my dotfiles (no system packages needed).

**Pull request closed without merging**